client/my-sites/plans/jetpack-plans/product-store/simple-product-card/index.tsx

```diff
@@ -21,6 +22,7 @@ const SimpleProductCard: React.FC< SimpleProductCardProps > = ( {
 
 	const { shortName: name } = item;
 	const productDescription = item?.shortDescription || item.description;
+	const hasMoreInfoLink = ! isDeprecated && ! isOwned && ! isIncludedInPlan;
```

Comment:
Instead of using this logic here, can we pass a prop like `hideMoreInfoLink` (`false` by default) and move this logic to the parent component?